java/midp/com/sun/cdc/io/j2me/file/DefaultFileHandler.java

```diff
@@ -0,0 +1,673 @@
+/*
```

Comment:
I guess this file is automatically generated from DefaultFileHandler.jpp, so shouldn't be added to the repo.